src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php

```diff
@@ -76,6 +77,26 @@ public function example(string|array $example): static
         return $this->attribute('example', $example);
     }
 
+    /**
+     * Sets the documentation URI, as usually put in the "@see" tag of a doc block. This
+     * can either be a URL or a file path. You can use the placeholders {package},
+     * {version:major} and {version:minor} in the URI.
+     *
+     * @return $this
+     */
+    public function docUrl(string $uri, ?string $package = null): static
+    {
+        if ($package) {
+            preg_match('/^(\d+)\.(\d+)\.(\d+)/', InstalledVersions::getVersion($package) ?? '', $m);
```

Comment:
No `$`, because some packages may return a fourth number (in local, `phpunit/phpunit` returns `9.6.99.0` for example on my computer)